script.js

```diff
@@ -0,0 +1,59 @@
+import submitForm from './modules/submit-form.js';
```

Comment:
Please change this file name to `index.js`.

<img width="163" alt="Screen Shot 2022-07-04 at 9 27 40" src="https://user-images.githubusercontent.com/82426682/177174628-9afbb585-14ae-44a5-86fa-d224fcdc2e90.png">